articles/virtual-machines/updates-maintenance-overview.md

```diff
@@ -9,90 +9,74 @@ ms.date: 02/28/2023
 ms.reviewer: cynthn
 ---
 
-# Updates and maintenance overview
+# Guest Updates and Host Maintenance Overview
```

Comment:
```suggestion
# Guest updates and host maintenance overview
```

Just an FYI that we use "sentence capitalization" so only the first word needs to be capitalized unless it is an approved brand name :)